systems/common/config/config.go

```diff
@@ -230,6 +238,58 @@ func LoadServiceHostConfig(name string) *Service {
 
 	return s
 }
+func LoadMailerHostConfig(name string) *Mailer {
```

Comment:
Do we have multiple services which would be using this LoadMailerHostConfig or it's just a mailer config ?